cli/src/pcluster/resources/compute_node/user_data.sh

```diff
@@ -151,6 +151,18 @@ write_files:
         done;
         export HOME="${!HOME_BAK}"
       }
+
+      function publish_startup_time
+      {
+        StartTime=$1
+        EndTime=$2
+        METRIC_NAMESPACE="ParallelCluster"
```

Comment:
Can we have a metric namespace/name cluster specific? To avoid mixing all the instances of all the clusters in the same metric. But please discuss this with @chenwany

Reply:
Hi Enrico, for `METRIC_NAMESPACE`, we discussed about either having `cluster_name` as namespace or as dimension. Here are the  reasons we use `ParallelCluster` as  namespace and `cluster_name` as dimensions:
* By setting cluster_name as an dimension, we are still able to aggregate data by cluster_name dimensions
* It provide more flexibility, we will be able to have statistic data with multiple clusters, for example, if this is used in our ops task, we can have the average startup time for all the clusters
* To keep consistency with the usage of other AWS metrics. For example, ECS cluster CapacityProviderReservation metric has AWS/ECS/ManagedScaling as namespace and cluster_name as dimension
* In our dev account, we are creating lots of cluster per day with integration test, in this case we will generate lots of Namespaces, they will be retained for some days/hours after the cluster get deleted. We can avoid large amount of namespaces to keep the console clean